test/tests/melt_pool_geometry/rosenthal_temp_source.i

```diff
@@ -0,0 +1,104 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmax = 2.0
+  ymin = -1
+  ymax = 1
+  nx = 2
+  ny = 2
+[]
+
+[Variables]
+  [temp]
+    initial_condition = 393
+  []
+[]
+
+[AuxVariables]
+  [temp_src]
+    order = FIRST
+    family = MONOMIAL
```

Comment:
why to use first monomial?

Reply:
For interpolating the material property and allowing contour plots.